src/data/uma-title-requirements.csv

```diff
@@ -1,11 +1,11 @@
 "text", "translation"
-"天気[雨]で4回以上出走し、3勝する","Participate a race with rainy weather 4 times or more\nWin at least 3 of them"
-"天気[曇り]で4回以上出走し、3勝する","Participate a race with cloudy weather 4 times or more\nWin at least 3 of them"
-"天気[雪]で3回以上出走し、2勝する","Participate a race with snowy weather 3 times or more\nWin at least 2 of them"
-"GⅠに1回以上出走し、\nGⅠ未勝利のまま育成ランクC以上で育成を完了する","<size=22>Participate a GI race once or more without winning any single GI race\nComplete the training with C or higher rank</size>"
+"天気[雨]で4回以上出走し、3勝する","Participate a race with rainy weather at least 4 times\nWin 3 of them"
+"天気[曇り]で4回以上出走し、3勝する","Participate a race with cloudy weather at least 4 times\nWin 3 of them"
+"天気[雪]で3回以上出走し、2勝する","Participate a race with snowy weather at least 3 times\nWin 2 of them"
+"GⅠに1回以上出走し、\nGⅠ未勝利のまま育成ランクC以上で育成を完了する","<size=22>Participate a GI race at least once and without winning any single GI race\nComplete the training with C or higher rank</size>"
 "ファン数が550000人以上になる","Earn a total of 550000 or more fans"
-"10回以上レースに出走し、\n芝のレースのみに出走して育成を完了する","<size=22>Participate a race 10 times or more\nEvery race has to be turf\nComplete the training</size>"
-"掛かったレースで3勝する","While under effect of temptation debuff\nWin a race 3 times"
+"10回以上レースに出走し、\n芝のレースのみに出走して育成を完了する","<size=22>Participate a race at least 10 times and every race has to be turf\nComplete the training</size>"
+"掛かったレースで3勝する","Win a race under panicked (掛かり) state 3 times"
```

Comment:
```suggestion
"掛かったレースで3勝する","Win a race under panicked (掛かり) state 3 times"
```
While actively still panicking while crossing the finish line or had panicked in the race and won regardless?

The sentence seem to say the former, but that seems hard to achieve and my gut says it's the latter 🤔

Reply:
its the latter

Reply:
if you have suggestion let me know

Reply:
Yes, I've been staring at this page for 5 minutes thinking about it 😂

Reply:
lol i know right

Reply:
Current one from bb44f17 looks good 👍